cluster/gce/gci/configure.sh

```diff
@@ -376,9 +376,18 @@ function try-load-docker-image {
   set +e
   local -r max_attempts=5
   local -i attempt_num=1
-  until timeout 30 ${LOAD_IMAGE_COMMAND:-docker load -i} "${img}"; do
+
+  if [[ "${CONTAINER_RUNTIME_NAME:-}" == "docker" ]]; then
+    load_image_command=${LOAD_IMAGE_COMMAND:-docker load -i}
+  elif [[ "${CONTAINER_RUNTIME_NAME:-}" == "containerd" ]]; then
+    load_image_command=${LOAD_IMAGE_COMMAND:-ctr -n=k8s.io images import}
+  else
+    load_image_command="${LOAD_IMAGE_COMMAND:-}"
```

Comment:
I think we shouldn't allow this else to succeed if the command is empty? maybe:
```suggestion
    load_image_command="${LOAD_IMAGE_COMMAND:?}"
```
or better yet an explicit check + error message

Reply:
the latter is probably best. `:?` will just ensure it was set.

Reply:
If it is unsupported runtime, it should fail at line 390. 
In theory, failing earlier will be better. But I did not see we
have functions like assert or exit in this file.